plugins/permission-backend/src/service/router.ts

```diff
@@ -136,17 +140,19 @@ export async function createRouter(
   router.post(
     '/authorize',
     async (
-      req: Request<Identified<AuthorizeRequest>[]>,
-      res: Response<Identified<AuthorizeResponse>[]>,
+      req: Request<AuthorizeRequestEnvelope>,
```

Comment:
Great call to not rely on array of objects, wrapping it in an object makes it easier to evolve over time.

I'm not sold on the Envelope suffix however, it really feels like something more lower level then the actual request/response. Would it be possible to keep `AuthorizeRequest` and `AuthorizeResponse` but those contain `items` instead?

Reply:
I do still need to do some local testing, but in the interest of gathering feedback sooner I just pushed 8726393, which renames types (and corresponding variables etc as far as I could find them) as follows:
- `AuthorizeRequestEnvelope` -> `AuthorizeRequest`
- `AuthorizeRequest` -> `AuthorizeQuery`
- `AuthorizeResponseEnvelope` -> `AuthorizeResponse`
- `AuthorizeResponse` -> `AuthorizeDecision`

PTAL! Seems worth spending a bit of time bikeshedding to avoid the risk that we wanna rename these again later. No need to delay today's release for this if the names turn out to be controversial!